rcl_action/src/rcl_action/goal_handle.c

```diff
@@ -26,6 +26,7 @@ typedef struct rcl_action_goal_handle_impl_s
 {
   rcl_action_goal_info_t info;
   rcl_action_goal_state_t state;
+  rcl_time_point_value_t goal_terminal_timestamp;
```

Comment:
There are some magic values that may be set, to indicate, that the value has not been set yet, this should be documented here,

Reply:
Okay.